### PR TITLE
WG Batch Annual report 2022

### LINKED ISSUE #7122: 2022 Annual Report: WG Batch
Chairs: @Huang-Wei @ahg-g @endocrimes @soltysh @swatisehgal 
Liaison: @mrbobbytables

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

A template has already been generated for your group, and is available [here](https://git.k8s.io/community/wg-batch/annual-report-2022.md).

Key dates:
- Initial PR to communtiy repo should be opened by March 24th, 2023
- PR should be reviewed and merged by April 7th, 2023

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/wg-batch/annual-report-2022.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/wg batch
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by soltysh**
Fixes #7122

/cc @Huang-Wei @ahg-g @endocrimes @swatisehgal 
/assign @mrbobbytables

### CODE REVIEW

**Review by mimowo — COMMENTED** [review]
Some ideas for expanding the work done by WG Batch in 2022.

File: wg-batch/annual-report-2022.md [hunk 1]
```diff
@@ -5,9 +5,8 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - We're still gathering feedback from multiple users running various batch-related frameworks and projects on kubernetes, see [our agenda](https://docs.google.com/document/d/1XOeUN-K0aKmJJNq7H07r74n-mGgSFyiEDQ3ecwsGhec/edit) for both past and upcoming topics.
+   - We've closely collaborated with SIG Apps, SIG Node and SIG Scheduling to improve batch workloads on kubernetes.
```
[the remark this answers is not in the document]
Comment (ahg-g) [reply wg-batch/annual-report-2022.md:9]: > I'm wondering what should go here:
> 
> * should we list KEPs? a couple made some graduation progress
> * should we list library / framework releases, for example Kueue
> * should we list presentation / discussion materials used during meetings? Maybe even the youtube recordings (I guess these could count as artifacts)

KEPs are already listed. But we can list releases of Kueue; I also think we can list presentations from kubecon:

- Introduction to Batch WG: https://www.youtube.com/watch?v=XeX2zBOykC4
- Kueue: A Kubernetes-native Job Queueing: https://www.youtube.com/watch?v=YwSZUdU3iRY&t=1s
- what else directly connected to the working group?

and perhaps mention that we organized two Batch Days in Kubecon EU and NA last year.

File: wg-batch/annual-report-2022.md [hunk 2]
```diff
@@ -5,9 +5,8 @@
...
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
```
Comment (alculquicondor) [inline wg-batch/annual-report-2022.md:11]: Development of Kueue, 2 releases in 2022

File: wg-batch/annual-report-2022.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (alculquicondor) [reply wg-batch/annual-report-2022.md:9]: I would highlight the keps to job.

Note that KEPs are not added to the working group report template. But we discussed pretty much every kep that involved job in the WG.

[the remark this answers is not in the document]
Comment (soltysh) [reply wg-batch/annual-report-2022.md:9]: Added:

File: wg-batch/annual-report-2022.md (hunk 2 above)
Comment (soltysh) [reply wg-batch/annual-report-2022.md:11]: Added

**Comment by soltysh** [comment]
@ahg-g @alculquicondor @mimowo I've addressed all the comments

**Review by alculquicondor — COMMENTED** [review]
LGTM as participant :)

**Comment by mimowo** [comment]
LGTM

File: wg-batch/annual-report-2022.md [hunk 3]
```diff
@@ -5,56 +5,48 @@
...
-- [ ] Updates provided to sponsoring SIGs in 2022
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      -
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
+- [x] Meeting notes and recordings for 2022 are linked from [README.md] and updated/uploaded if needed
+- [x] Updates provided to sponsoring SIGs in 2022
```
Comment (mrbobbytables) [inline wg-batch/annual-report-2022.md:49]:
```suggestion
- [x] Updates provided to sponsoring SIGs in 2022
  - [Introduction to Kubernetes WG Batch @ KubeCon EU 2022](https://youtu.be/XeX2zBOykC4)
```

Overall looks good :) 
Just wanted to include the vid from KubeCon EU, I dont think there was one for NA.

Comment (soltysh) [reply wg-batch/annual-report-2022.md:49]: Done

File: wg-batch/annual-report-2022.md [hunk 4]
```diff
@@ -5,56 +5,49 @@
...
+   - [Introduction to Batch WG during KubeCon EU 2022](https://www.youtube.com/watch?v=XeX2zBOykC4)
+   - Create [Kueue: A Kubernetes-native Job Queueing](https://www.youtube.com/watch?v=YwSZUdU3iRY) project and release [2 major versions in 2022](https://github.com/kubernetes-sigs/kueue/releases).
+   - Organize Batch Days during [KubeCon EU 2022](https://kubernetesbatchdayeu22.sched.com/) and [KubeCon NA 2022](https://kubernetesbatchdayna22.sched.com/)
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - Continuing work on [Kueue](https://github.com/kubernetes-sigs/kueue/releases).
```
[the remark this answers is not in the document]
Comment (soltysh) [reply wg-batch/annual-report-2022.md:18]: Added.

File: wg-batch/annual-report-2022.md [hunk 5]
```diff
@@ -5,56 +5,51 @@
...
+   - Continuing work on supporting specialized hardware vertical through [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
+     and [Scheduler plugins](https://github.com/kubernetes-sigs/scheduler-plugins).
```
Comment (swatisehgal) [inline wg-batch/annual-report-2022.md:20]: Thanks for incorporating my feedback. Just a minor rephrasing NIT:
```suggestion
   - Continuing work on Topology-aware Scheduling by enhancing [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
     and [Scheduler plugins](https://github.com/kubernetes-sigs/scheduler-plugins) to enable support for specialized hardware in Kubernetes.
```

Comment (soltysh) [reply wg-batch/annual-report-2022.md:20]: done

**Review by mrbobbytables — COMMENTED** [review]
/approve
Thanks!

**Pull request merged**